pkg/acsengine/engine.go

```diff
@@ -487,12 +487,16 @@ func getParameters(cs *api.ContainerService, isClassicMode bool) (map[string]int
 	return parametersMap, nil
 }
 
+func AddValue(m map[string]interface{}, k string, v interface{}) { addValue(m, k, v) }
```

Comment:
Updated in https://github.com/Azure/acs-engine/pull/957/commits/5777eff342c6bddae5aa5dd0e07fc69a092ab432